src/tests/unit/tests/injected/tabbable-element-getter.test.ts

```diff
@@ -64,4 +64,23 @@ describe('TabbableElementGetter', () => {
 
         expect(testSubject.get()).toEqual(expectedTabbleElementInfo);
     });
+
+    test('getRawElements', () => {
+        docMock.setup(m => m.documentElement).returns(() => documentElementStub);
+
+        const focusableElementsStub: FocusableElement[] = [
```

Comment:
nit: can we factor this out into a class constant so it doesn't have to be declared multiple times in the multiple tests?

Reply:
I think I'm going to remove the 'get' API completely actually, since it's not being used.

Reply:
I guess that will have to wait for Karan's PR since he's removing the usage of it.